practical/groovy/idr0021.groovy

```diff
@@ -0,0 +1,506 @@
+/*
+ * -----------------------------------------------------------------------------
+ *  Copyright (C) 2018 University of Dundee. All rights reserved.
+ *
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * ------------------------------------------------------------------------------
+ */
+
+/*
+ * This Groovy script uses ImageJ to analyse particles, and saved the results
+ * locally in a CSV file. The file is then uploaded and attached to the specified
+ * project as a file annotation.
+ * Use this script in the Scripting Dialog of Fiji (File > New > Script).
+ * Select Groovy as language in the Scripting Dialog.
+ * Error handling is omitted to ease the reading of the script but
+ * this should be added
+ * if used in production to make sure the services are closed
+ * Information can be found at
+ * https://docs.openmicroscopy.org/latest/omero/developers/Java.html
```

Comment:
``.../omero5/...`` is in https://github.com/ome/training-scripts/pull/31/files#diff-7b76ac0d9cb22c3981f59e92b5a80790R34 - why is it different here ? (``.../omero/...``)